src/components/map/MapContainer.jsx

```
@@ -28,17 +28,15 @@ const MapContainer = () => {
     setStartLocation,
   } = usePlaceStore();
-  const { isLoading, setIsLoading, setIsMapLoading } = useLoadingStore();
+  const { isLoading, setIsLoading } = useLoadingStore();
 
   const showError = (title, icon = 'error') => {
     Swal.fire({ title, icon });
   };
 
-  const waitForKakaoMaps = (callback, retries = 10) => {
+  const waitForKakaoMaps = (callback) => {
     if (window.kakao && window.kakao.maps) callback();
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)